src/test/suite/fixtures/recorded/languages/rust/chuckValueNear.yml

```diff
@@ -0,0 +1,32 @@
+languageId: rust
+command:
+  spokenForm: chuck value near
+  version: 2
+  targets:
+    - type: primitive
+      mark: {type: decoratedSymbol, symbolColor: default, character: 'n'}
+      modifiers:
+        - type: containingScope
+          scopeType: {type: value}
+  usePrePhraseSnapshot: true
+  action: {name: remove}
+initialState:
+  documentContents: |
+    let x = [None, Some(1)];
+  selections:
+    - anchor: {line: 0, character: 24}
+      active: {line: 0, character: 24}
+  marks:
+    default.n:
+      start: {line: 0, character: 9}
+      end: {line: 0, character: 13}
+finalState:
+  documentContents: |
+    let x = ;
```

Comment:
Is it valid to have eg `let x: u32` without an ` = ` (ie declaration without initialization)?  If so, I'd be tempted to remove the ` = ` when you "chuck value" here.  Would leave it for `"change"`

Reply:
You are right, but how do I achieve that?